src/pv.ts

```diff
@@ -208,10 +209,10 @@ export class PersistentVolume extends base.Resource implements IPersistentVolume
    *
    * @see https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reserving-a-persistentvolume
    */
-  public reserve(): pvc.PersistentVolumeClaim {
+  public reserve(namespace?: string): pvc.PersistentVolumeClaim {
```

Comment:
We don't normally use optional positional arguments. Instead, create a `PersistentVolumeReserveOptions` interface.

Also this requires a test.